payload/wwfcFeature.cpp

```diff
@@ -218,6 +218,35 @@ WWFC_DEFINE_PATCH = {
     ),
 };
 
+// Report information about the upcoming match to the server
+WWFC_DEFINE_PATCH = {
+    Patch::CallWithCTR( //
+        WWFC_PATCH_LEVEL_FEATURE, //
+        RMCXD_PORT(0x8066148C, 0x80659550, 0x80660AF8, 0x8064F7A4), //
+        // clang-format off
+        []() -> void {
+            using namespace mkw::Net;
+
+            SelectHandler* selectHandler = SelectHandler::Instance();
+            selectHandler->decideCourse();
+            selectHandler->initPlayerIdsToPlayerAids();
+
+            SelectHandler::Packet::SelectedCourse selectedCourse =
+                selectHandler->sendPacket().selectedCourse;
+            SelectHandler::Packet::EngineClass engineClass =
+                selectHandler->sendPacket().engineClass;
+
+            wwfc::GPReport::ReportB64Encode(
```

Comment:
Why encode with base64 here instead of casting to a u32 and using `ReportU32`?

Reply:
Addressed in commit [ca11ee0](https://github.com/WiiLink24/wfc-patcher-wii/pull/67/commits/ca11ee06626e5f43594504eb69701c7b50c38a55).